src/ssort/_main.py

```diff
@@ -58,7 +59,9 @@ def main():
     paths = _find_files(args.files)
     for path in paths:
         errors = False
-        original = path.read_text()
+        original_bytes = path.read_bytes()
+        encoding = detect_encoding(original_bytes)
+        original = original_bytes.decode(encoding)
```

Comment:
An invalid encoding should probably have the same behavior as a syntax error. The behavior here is to immediately terminate and skip any files which come after the file with the invalid encoding. I would think this is undesirable.

Reply:
Good point.  Will fix.